app/models/site.rb

```diff
@@ -411,7 +411,7 @@ def default_threshold_for_moderation_delay
     end
 
     def default_threshold_for_referral
-      ENV.fetch('THRESHOLD_FOR_REFERRAL', '50').to_i
+      ENV.fetch("THRESHOLD_FOR_REFERRAL", "250").to_i
```

Comment:
I don't think we actually need to change the default, but just thought it might make sense to have a default that reflects the live value.